workbench-core/example/infrastructure/package.json

```diff
@@ -18,9 +18,14 @@
     "build:test": "rushx compile && heft test -u --clean && rushx pkg-json-lint",
     "cdk": "cdk",
     "cdk:bootstrap": "cdk bootstrap",
-    "cdk:deploy": "rush build -f @aws/workbench-core-example-infrastructure && rushx compile && rushx cdk deploy --all --require-approval never --outputs-file src/config/testEnv.json",
-    "cdk:deploy-noBuild": "rushx compile && rushx cdk deploy --all --require-approval never --outputs-file src/config/testEnv.json",
-    "cdk:destroy": "rushx cdk destroy --all",
+    "cdk:bootstrap:hosting": "cdk bootstrap aws://$HOSTING_ACCOUNT_ID/$HOSTING_ACCOUNT_REGION --trust $MAIN_ACCOUNT_ID --trust-for-lookup $MAIN_ACCOUNT_ID --cloudformation-execution-policies 'arn:aws:iam::aws:policy/AdministratorAccess'",
+    "cdk:bootstrap:main": "cdk bootstrap aws://$MAIN_ACCOUNT_ID/$MAIN_ACCOUNT_REGION",
```

Comment:
these commands dont work with Windows

Reply:
Neither git bash nor powershell

Reply:
What error do you get ? These commands are just using the environment variables declared before. How do you set environment variables in git bash or powershell ?

Reply:
The command doesnt take in the env variables. It takes the `$whatever` variables as strings. So it tries to run `cdk bootstrap aws://$MAIN_ACCOUNT_ID/$MAIN_ACCOUNT_REGION` rather than `cdk bootstrap aws://1234567890/us-east-1`

Reply:
env variables are set using `export VARIABLE=whatever` in git bash and `$env:VARIABLE=whatever` in powershell

Reply:
the use of variables with $ is valid in powershell: https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_variables?view=powershell-7.3

Reply:
The issue was with the rush shortcut, running `cdk bootstrap aws://$MAIN_ACCOUNT_ID/$MAIN_ACCOUNT_REGION` in git bash works

Reply:
Added a note for Windows users